### PR TITLE
add vmware 0.5.4 to index

### PR DESCRIPTION
**Pull request by ctaggart**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by ctaggart** [comment]
CI failed with `You have CLI core version 2.2.0 and this extension requires a min of 2.0.66 and max of 2.1.0.`

**Comment by ctaggart** [comment]
Removing it didn't work:
```
ERROR: The 'None' extension is not compatible with this version of the CLI.
You have CLI core version 2.2.0 and this extension requires a min of 2.0.66 and max of 2.1.0.
```
Do I have to update to the latest CLI core to get it past this?

**Comment by ctaggart** [comment]
CI passes! :tada:

**Comment by qwordy** [comment]
It's a long extension name. 😂

**Comment by ctaggart** [comment]
Yes, `azure-vmware-virtustream-cli-extension` is a long name. Would you like it renamed?

**Comment by ctaggart** [comment]
`az extension add --name azure-vmware-virtustream-cli-extension` isn't the most convenient. 
`az extension add --name vmware` would be nice.

**Comment by ctaggart** [comment]
Hold off on merging this. I'm going to change it to just `vmware`.

**Comment by shivam-msft** [comment]
> Hold off on merging this. I'm going to change it to just `vmware`.

Sure Cameron. Just "vmware" would be nice.

**Comment by ctaggart** [comment]
The extension installs and works just fine with:
```
az extension add -s https://github.com/virtustream/azure-vmware-virtustream-cli-extension/releases/download/0.5.3/vmware-0.5.3-py2.py3-none-any.whl -y
```

CI then runs azdev linter and fails. Any ideas why?
```
azdev linter --include-whl-extensions vmware
Traceback (most recent call last):
  File "scripts/ci/verify_linter.py", line 213, in <module>
    main()
  File "scripts/ci/verify_linter.py", line 195, in main
    linter_on_external_extension(modified_files[0])
  File "scripts/ci/verify_linter.py", line 160, in linter_on_external_extension
    azdev_extension.linter()
  File "scripts/ci/verify_linter.py", line 77, in linter
    self._cmd('azdev linter --include-whl-extensions {}'.format(self.extension_name))
  File "scripts/ci/verify_linter.py", line 68, in _cmd
    check_call(cmd, shell=True)
  File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/subprocess.py", line 311, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'azdev linter --include-whl-extensions vmware' returned non-zero exit status 1.
```

**Comment by shivam-msft** [comment]
> The extension installs and works just fine with:
> 
> ```
> az extension add -s https://github.com/virtustream/azure-vmware-virtustream-cli-extension/releases/download/0.5.3/vmware-0.5.3-py2.py3-none-any.whl -y
> ```
> 
> CI then runs azdev linter and fails. Any ideas why?
> 
> ```
> azdev linter --include-whl-extensions vmware
> Traceback (most recent call last):
>   File "scripts/ci/verify_linter.py", line 213, in <module>
>     main()
>   File "scripts/ci/verify_linter.py", line 195, in main
>     linter_on_external_extension(modified_files[0])
>   File "scripts/ci/verify_linter.py", line 160, in linter_on_external_extension
>     azdev_extension.linter()
>   File "scripts/ci/verify_linter.py", line 77, in linter
>     self._cmd('azdev linter --include-whl-extensions {}'.format(self.extension_name))
>   File "scripts/ci/verify_linter.py", line 68, in _cmd
>     check_call(cmd, shell=True)
>   File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/subprocess.py", line 311, in check_call
>     raise CalledProcessError(retcode, cmd)
> subprocess.CalledProcessError: Command 'azdev linter --include-whl-extensions vmware' returned non-zero exit status 1.
> ```

Hi @qwordy ,

As far as I understand, there are some new linter verifications which were merged from master in this PR (in this [commit](https://github.com/Azure/azure-cli-extensions/commit/ff1356ade9e12c638d3aa0437d9c175d079e608e#)) which made the CI fail. Can you have a look.

**Comment by haroldrandom** [comment]
@shivam-msft I am taking care of it.

**Comment by ctaggart** [comment]
I'm going to bump this up to 0.5.4 as requested.

**Comment by haroldrandom** [comment]
@ctaggart The bug in job "CLI Linter on Modifed Extensions" has been fixed. It's feel to go. please take care of the rest of problems.

Welcome to ping me if anything wrong in CI.

File: src/index.json [hunk 1]
```diff
@@ -3006,6 +3006,40 @@
...
+                                "Home": "https://github.com/virtustream/azure-vmware-virtustream-cli-extension"
+                            }
+                        }
+                    },
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "vmware",
+                    "summary": "Preview Azure VMware Solution by Virtustream commands.",
+                    "version": "0.5.3"
```
Comment (haroldrandom) [inline src/index.json]: The version should be 0.5.4

**Comment by ctaggart** [comment]
Thanks @haroldrandom !

**Comment by ctaggart** [comment]
CI is passing again. All requested updates are complete. This is ready for a squash merge. cc @shivam-msft

**Comment by yungezz** [comment]
will merge it after sync with master branch

**Comment by shivam-msft** [comment]
> CI is passing again. All requested updates are complete. This is ready for a squash merge. cc @shivam-msft

Looks good to me @ctaggart

**Comment by ctaggart** [comment]
> will merge it after sync with master branch

@yungezz I merged the master branch into this repository and it built fine. Is that what you meant? Can you merge this pull request?

**Comment by ctaggart** [comment]
It works! Here is a screenshot from Azure Cloud Shell:

image

**Comment by shivam-msft** [comment]
> It works! Here is a screenshot from Azure Cloud Shell:
> 
> image

Awesome!

**Pull request merged**